app/workers/adobe_campaign_worker.rb

```diff
@@ -73,11 +73,17 @@ def email_address
     @email_address ||= params[email_address_name]&.downcase
   end
 
+  def prefer_not_to_say(key, value)
+    return true if key == "Country" || key == "state" && value == "AA"
+    false
```

Comment:
```suggestion
  def prefer_not_to_say?(key, value)
    %w[Country state].include?(key) && value == "AA"
```

...rubocop might suggest similar